Sample/freshchat_utils.js

```diff
@@ -0,0 +1,21 @@
+import {
+    Freshchat,
+    FreshchatConfig,
+    FaqOptions,
+    ConversationOptions,
+    FreshchatUser,
+    FreshchatMessage,
+    FreshchatNotificationConfig
+} from 'react-native-freshchat-sdk';
+
+appId = 'c69641e9-8a85-4da1-858e-77169b0c76a7';
```

Comment:
Should we include sensitive info here in the code, or should we pass it via environment variables?

Reply:
Removed all sensitive informations. Thanks.